tests/backwards_compatibility_tests/base_test_case.py

```diff
@@ -0,0 +1,58 @@
+import json
+import logging
+import os
+import sys
+from pathlib import Path
+
+from marqo_test import MarqoTestCase
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
+
+class BaseCompatibilityTestCase(MarqoTestCase):
```

Comment:
I think this is an ABC, and prepare(self) is a `@abstractmethod`, right?

Reply:
done, https://github.com/marqo-ai/marqo/pull/1024/commits/740097f76bc5fd67568d409c1a680d9ebcb77333#diff-3b756b38875ef9e366500400a91b87df9b4a91e7a547e2de0a1886fcc605ca50R55